src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
[Don't forget how to name constants according to style guide (the same with naming of enum values)](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values)
```suggestion
public enum Color {
```